pkg/webhook/webhook.go

```diff
@@ -199,7 +195,90 @@ func (wh *webhook) validate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRespo
 			Status: metav1.StatusFailure, Code: http.StatusForbidden, Reason: "PVC with cloned volumes can't be deleted",
 			Message: fmt.Sprintf("pvc %q has '%v' cloned volume(s)", pvc.Name, len(cStorVolumes.Items)),
 		}
+		return response
+	}
+	return response
+}
+
+// validatePVCCreateRequest validates persistentvolumeclaim(PVC) create request
+func (wh *webhook) validatePVCCreateRequest(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
```

Comment:
Please provide the UT for this logic

Reply:
did we added UT?

Reply:
ok.. got to see UT.. but, failure cases are not covered. Would that be possible to cover?